tests/test_churn_classification/test_churn_classification.py

```diff
@@ -36,13 +36,16 @@ def test_development_sql_repo(self, init_sql_repo):
         self.waiting_jobs_to_complete = False
         self.__run()
 
+    @pytest.mark.churn_fs
     def test_standalone_fs_repo(self):
-        Config.configure_job_executions(mode=JobConfig._STANDALONE_MODE, max_nb_of_workers=2)
+        Config.configure_job_executions(mode=JobConfig._STANDALONE_MODE, max_nb_of_workers=4)
         self.waiting_jobs_to_complete = True
         self.__run()
 
+    @pytest.mark.churn_sql
+    @pytest.mark.skip("This test is skipped because it is not working as expected.")
```

Comment:
Should check why it is skipped.

Reply:
This is intermitent, because the iteraction of the standalone mode with SQL.